src/org/omegat/core/dictionaries/LingvoDSL.java

```diff
@@ -220,5 +222,7 @@ private static String replaceTag(final String line) {
         for (String tag : ignoreTags) {
             TAG_REPLACEMENTS.put(Pattern.compile("\\[(?<tag>" + tag + ")](?<content>.+?)\\[/\\k<tag>]"), "${content}");
         }
+        // remove orphan trn tag
+        TAG_REPLACEMENTS.put(Pattern.compile("\\[trn]"), "");
```

Comment:
Have we seen orphan `trn` in the wild? Do we know why only `trn` needs this treatment?

Reply:
Susan report it in omegat-users ML. I see it in capture.

Reply:
Any tag of `ignoreTags` may be able to be orphan.